salt/edx/templates/ansible_env_config.yml.j2

```diff
@@ -115,8 +115,8 @@ lms_auth_config:  &lms_auth
       password: "{{ MONGODB_CONTENTSTORE_PASSWORD }}"
       port: "{{ MONGODB_PORT }}"
       user: "{{ MONGODB_CONTENTSTORE_USER }}"
-    ADDITIONAL_OPTIONS:
```

Comment:
Why is this not required anymore? I think ADDITIONAL_OPTIONS is passed on as the kwargs to the pymongo constructor in the modulestore.